draft-ietf-rats-corim.md

```diff
@@ -1239,7 +1240,22 @@ applies to all measurements in the triple, including those in `measurement-value
 
 ## Extensibility {#sec-extensibility}
 
-[^issue] https://github.com/ietf-rats-wg/draft-ietf-rats-corim/issues/10
+CoRIM is described using CDDL {{-cddl}} which includes extensibility support for "extension points" (a.k.a., map structures)
```

Comment:
The whole structure of the section needs reorganising. I will suggest a better rewrite as an edit to this section!